README.adoc

```diff
@@ -10,7 +10,7 @@ image:https://img.shields.io/badge/license-Apache2-blue.svg["Apache 2.0 license"
 
 Kiali provides answers to the question _What are the microservices in my Istio service mesh doing?_
 
-image::https://www.kiali.io/images/features/feature3.png[Kiali Graph]
+image::kiali-full-ui.png[Kiali Graph, width=480]
```

Comment:
Have you considered linking to the images in the kiali.io github repo? that would avoid duplicating and maintaining the image twice.
For instance linking here:
https://github.com/kiali/kiali.io/blob/master/static/images/gettingstarted/kiali-graph.png
or here:
https://github.com/kiali/kiali.io/blob/master/static/images/features/graph-overview.png

Reply:
That's how it was done in the past.  But when we made changes to kiali.io, we inadvertently broke the link.  As a result we have this current issue. Your thoughts?

Reply:
if I understand correctly, before this PR, the linking was done directly to the image on the website itself (kiali.io). What I am proposing is linking to the image on github repo of kiali.io, not to the website itself.

Reply:
Ok I misunderstood. I can push an update with a link to graph-overview.png